README.md

```diff
@@ -44,7 +46,17 @@
    - [Queries](#queries)
    - [Event Creation](#event-creation)
 
-8. [About Us](#about-us)
+10. [About Us](#about-us)
+
+## Project Intro
+
+- Dot to Dot is a community-focused organization dedicated to reconnecting communities with local makers and artisans. The Dot to Dot booking system aims to create inspiring events that showcases talents in the local community. Through a specialised log-in system, this allows users to have a personalised experience, to track bookings whilst also allowing artist to create their own unique events with the help of an additional admin page.
+
+## Useful Links
+
+- Live site - https://union-client-project.web.app/splash
```

Comment:
With markdown the syntax for adding a link is to use `[]()`
e.g. Writing `[Live site](https://union-client-project.web.app/splash)` will give you [Live site](https://union-client-project.web.app/splash)